src/auditlog/models.py

```diff
@@ -94,6 +94,9 @@ def get_for_objects(self, queryset):
 
         if isinstance(primary_keys[0], integer_types):
             return self.filter(content_type=content_type).filter(Q(object_id__in=primary_keys)).distinct()
+        elif isinstance(queryset.model._meta.pk, models.UUIDField):
```

Comment:
Another improvement of checking `queryset.model._meta.pk` and not `primary_keys[0]` is that `primary_keys[0]` will error raise an `IndexError` if there are no records for the table where checking `queryset.model._meta.pk` will not.